4_params_munge/src/munge_bathy.R

```diff
@@ -28,6 +28,19 @@ munge_simple_depth <- function(out_ind, ...){
   gd_put(out_ind, data_file)
 }
 
+munge_navico_depths <- function(out_ind, navico_xwalk_ind) {
+  # munge depth data and filter out lakes where max depth is NA
+  navico_depths <- sc_retrieve(navico_xwalk_ind) %>%
+    readRDS() %>%
+    mutate(Navico_ID = sprintf("Navico_%s", Navico_ID)) %>%
```

Comment:
This PR looks good to me. I'm just wondering if there is a reason to prefix the individual IDs with Navico here in 4_ instead of within the `rename(site_id = MapWaterbody_ID) %>%` in `fetch_navico_points()` back in 1_ so that all of the Navico IDs are prefixed with "Navico"?

Reply:
Oh good thinking - I'll make that change